includes/Admin/Orders.php

```diff
@@ -159,6 +159,18 @@ public function handle_bulk_update() {
 	 */
 	public function maybe_stop_order_email( $is_enabled, $order ) {
 
+		$is_enabled = $is_enabled && ! ( $order instanceof \WC_Order && \SkyVerge\WooCommerce\Facebook\Commerce\Orders::is_commerce_order( $order ) );
+
+		/**
+		 * Filters the flag used to determine whether the email is enabled.
+		 *
+		 * @since 2.1.0-dev.1
+		 *
+		 * @param bool $is_enabled whether the email is enabled
+		 * @param Orders $this admin orders instance
+		 */
+		$is_enabled = (bool) apply_filters( 'wc_facebook_commerce_send_woocommerce_emails', $is_enabled, $this );
```

Comment:
I think we'll want to scope this filter to only apply if the order is a commerce order and `$is_enabled` was previously true. The name implies that it would only affect commerce orders, but filtering this to `true` would also enable these emails for regular orders, even if they are disabled in settings.

Also, perhaps we can pass the order as an additional filter param?

Reply:
> I think we'll want to scope this filter to only apply if the order is a commerce order and $is_enabled was previously true. The name implies that it would only affect commerce orders, but filtering this to true would also enable these emails for regular orders, even if they are disabled in settings.

Yeah, that makes sense and it's fixed in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1567/commits/3e69335932ad8ee7991ed25415d8360123b12d32 and https://github.com/facebookincubator/facebook-for-woocommerce/pull/1567/commits/7902bd26959a0eb01a7919b6e2d99f2259b78358.

In `maybe_stop_order_email` a more elegant solution IMO would be starting with

```
if ( ! $is_enabled ) {
    return false;
} 
```

However, considering that early return is not a good practice, I decided to make use of a `$is_previously_enabled` var.

> Also, perhaps we can pass the order as an additional filter param?

Done in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1567/commits/6f4284d20de1a5a0408d68a9ed55e49e73387c6a.